cost/aws/schedule_instance/aws_schedule_instance.pt

```diff
@@ -413,7 +508,14 @@ script "js_instances_with_schedule", type: "javascript" do
       })
     }
 
-    if (schedule != null) {
+    // Check that we have a schedule and it is not explicitely disabled before attempting to parse the schedule tag value
+    // Test the regex pattern for a valid schedule as well, in case the tag value is not a valid schedule
+    // schedule="08:00-17:30;MO,TU,WE,TH,FR"
+    // schedule="08:00-17:30;MO,TU,WE,TH,FR;America/Los_Angeles"
+    var schedule_is_valid = /([0-9]{1,2}.*)-([0-9]{1,2}.*);([A-Za-z,]*).*/.test(schedule)
+    if ((_.isString(schedule)) && (schedule_is_valid == false)) {
+      console.log("Skipping instance " + instance['instanceId'] + " with invalid schedule '" + schedule +"'");
```

Comment:
Do we need this console.log for this template meant for the Catalog?

Reply:
Using this thread to discuss:
https://github.com/flexera-public/policy_templates/pull/2817#discussion_r1834542127